src/core/server/errors/index.ts

```diff
@@ -259,11 +259,7 @@ export class URLInvalidError extends CoralError {
 }
 
 export class StoryURLInvalidError extends CoralError {
-  constructor(properties: {
-    storyURL: string;
-    allowedDomains: string[];
-    tenantDomain?: string;
-  }) {
+  constructor(properties: { storyURL: string; tenantDomain?: string }) {
```

Comment:
Revisit if we can include this

Reply:
so the way we find out if a URL is valid has changed a bit, previously we would check tenant.allowedDomains, and then create the story only if the ULR matched one of the domains, and throw this error otherwise. The new approach is to look for a site where site.allowedDomains matches the URL, if there's no match, it returns null. To populate `allowedDomains` on the error message we'd have to query all sites and then concat all the site.allowedDomains together, which we can definitely do, if it will be valuable for debugging in prod.